modular_skyrat/modules/medical/code/smartdarts.dm

```diff
@@ -57,10 +63,11 @@
 	projectile_type = /obj/projectile/bullet/dart/syringe/dart
 
 /obj/projectile/bullet/dart/syringe/dart
-	name = "smartdart"
+	name = "SmartDart"
 	damage = 0
 	var/list/allergyList = list()
-	var/prevention_used = FALSE
+	var/prevention_used = FALSE //Is allergy prevention used?
```

Comment:
AUTODOC documentation always comes on the line before the variable, and is indicated by `///`, not just `//`